src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/MicrosoftIdentityPlatform/AppSettingsModifier.cs

```diff
@@ -9,6 +9,9 @@ namespace Microsoft.DotNet.MSIdentity.MicrosoftIdentityPlatform
 {
     internal class AppSettingsModifier
     {
+        private const string MicrosoftGraph = "MicrosoftGraph";
+        private const string DownstreamApi = "DownstreamApi";
+        private const string ServerApi = "ServerApi";
```

Comment:
I definitely prefer `nameof(VarName)` instead of `VarName="VarName"` if the variable name is the exact same, just less chances for the string to be accidentally edited, i would change all these.

Reply:
Cool sounds good